src/renderers/shared/fiber/ReactFiberCompleteWork.js

```diff
@@ -201,7 +201,15 @@ module.exports = function<T, P, I, TI, PI, C, CX, PL>(
           // Even better would be if children weren't special cased at all tho.
           const instance : I = workInProgress.stateNode;
           const currentHostContext = getHostContext();
-          const updatePayload = prepareUpdate(instance, type, oldProps, newProps, rootContainerInstance, currentHostContext);
+          const updatePayload =
+            prepareUpdate(
+              instance,
+              type,
+              oldProps,
+              newProps,
+              rootContainerInstance,
+              currentHostContext);
```

Comment:
nit: `);` on new line is more readable (eg see [here](https://github.com/facebook/react/blob/3c75593692524678fbb74e8077018cc39b04ffaa/src/renderers/shared/fiber/ReactFiberCompleteWork.js#L243))